runtime/makelib/targets.mk.ftl

```diff
@@ -557,7 +560,10 @@ NEW_OPTIMIZATION_FLAG+=-qmaxmem=-1 -qpic
 </#if>
 SPECIALCXXFLAGS=$(filter-out $(FLAGS_TO_REMOVE),$(CXXFLAGS))
 
-BytecodeInterpreter$(UMA_DOT_O):BytecodeInterpreter.cpp
+BytecodeInterpreterFull$(UMA_DOT_O):BytecodeInterpreterFull.cpp
+	$(CXX) $(SPECIALCXXFLAGS) $(NEW_OPTIMIZATION_FLAG) -c $<
+
+BytecodeInterpreterCompressed$(UMA_DOT_O):BytecodeInterpreterCompressed.cpp
 	$(CXX) $(SPECIALCXXFLAGS) $(NEW_OPTIMIZATION_FLAG) -c $<
```

Comment:
Agreed - do you know offhand how to do this? I don't want to have to mess with makefiles (more than I already do) in order to mixify a build. If it can be done based on the flags in `omrcfg.h`, then great. Otherwise, I'd prefer to either ignore the OSX issue for now, or change the files to contain a dummy static or function when not defining the custom interpreter to work around the issue.

Reply:
I suppose we can't ignore OSX when we publish these changes, so the dummy code is the short-term answer if you can't fix the makefiles cleanly.

Reply:
Going with the temporary solution for now - dummy define in the split interpreter files. Will look into makefile changes in future PR.